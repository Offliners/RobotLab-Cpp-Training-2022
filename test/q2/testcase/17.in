106
EDITION
mELAncholy
CONVINCE
MiRROR
DonAte
DINE
LIQUID
SERUM
FuRtHeR
EIGHT
attitude
resORT
DeFeND
QUALIFY
NeGOtiATe
tireD
eNCyCLOPEDIA
humOROUS
ORNAMENT
loyalty
dROWSy
almoST
THeOLogicAL
diet
welfare
matter
lEttEr
KINGDOM
MiscellAneOUs
PlEDGE
reflecT
ConStant
upgradE
cURb
acCUsAtIoN
pHYSICS
InTErFERENce
bUrEau
Mighty
depletE
INSPEct
DISTRUST
rEfrIgERator
aGAINSt
THEOLOGIAN
UNUSUALLy
COllABORaTe
Refinement
GReaSE
doUGHnut
APPROPRiatE
RUB
ECoNOMIC
sCIENtIst
SNAKE
comiC
LeT
prohibition
ALcOHoL
iMAGINE
accumulate
LiNgeRIng
Champagne
STABLE
ADMiniSTRatIve
diffeRence
GreAtNESs
signal
be7ut254
resemblance
cOlUmN
BLESS
huRT
homEland
WeST
counTaBle
AlTHOUGH
ShOCKINg
COCONUT
mOnUMEnT
invEstiGaTe
HUMbly
portray
ArRAy
DOrMITOrY/dorM
PIcTUreSQuE
kNIGhT
upSTaiRs
HyPOcRISy
EXIT
cOMPoSItION
CONJUNCTION
TrAffic
biologIcal
COMPOSITE
draMa
Rare
diShonesT
HALFTVay
SITUATION
eVentual
ElEctrIcIaN
pitch
sOMEHOW
ExPANSION
CHEER
