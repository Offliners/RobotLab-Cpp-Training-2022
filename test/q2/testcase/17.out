n. 發行數；版本
n./adj. 悲哀；憂鬱
v. 說服
n./v. 鏡子；反映
v. 捐贈
v. 用餐
n. 液體
n. 血清
adj./adv./v. 進一步的
adj. /n八
n. 態度
v. 前往
v. 防守；防衛
v. 使具有資格
v. 談判
adj. 疲倦的
n. 百科全書
adj. 幽默的
n. 裝飾
n. 忠誠
adj. 昏昏欲睡的
adv. 幾乎
adj. 神學(上)的
n. 飲食；特種飲食
n. 福利
n. 事情；問題
n. 信；字母
n. 王國；王權
adj. 各式各樣的
v. 發誓；保證
v. 反射
adj. 固定的
v. 提升
n. 抑制
n. 控訴
n. 物理學
n. 阻礙；干預
n. 局；處；署
adj. 強大的；偉大的
v. 使枯竭
v. 檢查
v. 不信任
n. 冰箱
prep. 反對；對著
n. 神學家
adv. 非常
v. 合作
n. 精製品
n. 油脂
n. 甜甜圈
adj. 適當的
v. 摩擦
adj. 經濟上的
n. 科學家
n. 蛇
adj. 滑稽的
v. 允許；讓
n. 禁止
n. 酒精
v. 想像
v.  累積
adj. 持久的
n. 香檳
adj. 穩定的
adj. 行政的
n. 差異
n. 很多
n./v. 信號
Unknown
n. 相似
n. 欄；圓柱
v./n. 為…祝福
n./v. 受傷
n. 祖國；家夠卜
adj./adv./n. 西方的
adj. 可數的
conj. 雖然；然而
adj. 受震驚的
n. 椰子
n. 紀念碑；紀念館
v. 調查；研究
adv. 謙虛地
v. 描繪
n. 一長列
n. 學生宿舍
adj. 似畫的
n. 騎士；爵士
adj. 在樓上
n. 虛偽
n./v. 出口
n. 作文；合成物
n. 結合
n. 交通
adj. 生物(學)的
adj. 混合成的
n. 戲劇
adj. 珍貴的
adj. 不誠實的
adj./adv. 位於兩地中途的
n. 形勢
adj. 結果的
n. 電工；電氣技師
n. 瀝青
adv. 不知怎麼地
n. 擴展；膨脹
n./v. 歡呼；激勵
